torchtune/modules/peft/lora.py

```diff
@@ -65,7 +65,7 @@ def __init__(
         self.use_bias = use_bias
         self._quantize_base = quantize_base
 
-        if not self._quantize_base and quantization_kwargs:
+        if not self._quantize_base and any([v for v in quantization_kwargs.values()]):
```

Comment:
did `and quantization_kwargs` not work for a dictionary? you could also just check for length

Reply:
```
In [1]: any([None])
Out[1]: False
```

Reply:
sorry `{use_lora_on_output: None}` should fail the check?

Reply:
If quantize_base is false, then we should NOT have any quantization args, right? If no quantization is happenning, then why have these args?

So this assertion was failing, because quantize_base WAS false, so not False --> True, but we have {use_lora_on_output: None}, which returns True.

So if True and True: raise error.

Reply:
I agree with Rafi here, there shouldn't be any kwargs passed in if quantize_base is False. {use_lora_on_output: None} should not be passed in when quantize_base = False.

Reply:
<img width="323" alt="image" src="https://github.com/user-attachments/assets/8c75d8e6-2ed8-42e7-ad87-e3c393e31b27">